simplq/src/components/pages/Admin/ControlScroll.js

```diff
@@ -0,0 +1,44 @@
+// left: 37, up: 38, right: 39, down: 40,
+// spacebar: 32, pageup: 33, pagedown: 34, end: 35, home: 36
+var keys = {37: 1, 38: 1, 39: 1, 40: 1};
+
+function preventDefault(e) {
+  e.preventDefault();
```

Comment:
Again, just for consistency's sake, can we stick to using arrow functions everywhere instead of declaring with the function keyword?